packages/wdio-appium-service/appium-service.d.ts

```diff
@@ -1,7 +1,24 @@
+/// <reference types="webdriver"/>
+/// <reference types="webdriverio"/>
+
```

Comment:
It doesn't seem we use any other WebDriver or WebdriverIO references here, so we can get rid of it.
```suggestion
```

Reply:
removed